### ISSUE TITLE
Duckstation crashes with segmentation fault on Redmi 4A when using OpenGL render

### ISSUE THREAD

**Issue by Jenmet-cy** [stenzek/duckstation#961]
Using DS with OpenGL rendering causes a crash in the emulator with segmentation fault on Redmi 4A. Software rendering works without issues, though is a bit slow. The phone has OpenGL ES 3.0 support, so there shouldn't be any issues. Here is my logcat:

**Comment by stenzek** [comment]
Looks like some GL function might be missing.. but from that backtrace it's difficult to say which one. Would have to use a debug build.

As an aside, GLES3.0 is going to be quite limiting - it won't use the efficient texture upload path, you need GLES3.2 for that.

**Comment by Jenmet-cy** [comment]
Ah, I didn't know that GLES3.0 was limiting. Where can I get a debug build to send you a more detailed logcat?

**Comment by stenzek** [comment]
Should hopefully be fixed with https://github.com/stenzek/duckstation/commit/19f4921ca78d6c8d789647da2a73477e592ea4a7 - will push to the play store in the next couple of days assuming nobody else reports any other errors.

**Comment by Jenmet-cy** [comment]
Yup, the hardware renderer is working now on my end. Using the latest dev build.
![Screenshot_20201014-111431_DuckStation](https://user-images.githubusercontent.com/52474680/96001646-a2e24180-0e0e-11eb-937a-6fe83c4065f0.png)
![Screenshot_20201014-111857_DuckStation](https://user-images.githubusercontent.com/52474680/96002071-197f3f00-0e0f-11eb-91c6-44a2d8f4e1c8.png)

**Comment by stenzek** [comment]
Great, thanks for checking!

Without dual-source blend you'll probably find there's a few broken games (particularly those which use the mask bit). But most should be okay.

(also, the speed is a bit low, but kinda expected for a 1.4GHz Cortex-A53. I have some performance improvements coming soon which will hopefully be enough to bring it up to full speed).

**Comment by Jenmet-cy** [comment]
Neat! Thanks to you for fixing it. Looking forward to the performance improvements.
